tools/afl/src/readline.c

```diff
@@ -16,21 +16,28 @@
 
 #include "readline.h"
 
+#include <errno.h>
 #include <stdlib.h>
 
+#define MAX_SANE_LENGTH (50 * 1024 * 1024)
```

Comment:
max 50 MBytes, right?

Reply:
Yeah, 50 MB. This is default memory limit of AFL and our test data is unlikely to exceed it. There is no gain in making a huge test case as AFL will try minimizing it anyway.

Reply:
makes sense, thanx!